tests/integration_tests/capture_session_done_during_task/run.sh

```
@@ -56,7 +56,8 @@ function run() {
 
 	# ensure server exit
 	ensure 30 "! ps -p $cdc_pid"
-	check_logs_contains $WORK_DIR "the etcd session is done"
+	check_logs_contains $WORK_DIR "local fence triggered"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)